lsp-mode.el

```diff
@@ -4065,7 +4065,9 @@ PLIST is the additional data to attach to each candidate."
                          "textEdit" text-edit
                          "insertTextFormat" insert-text-format
                          "additionalTextEdits" additional-text-edits)
-                  (lsp--resolve-completion item)))
+                  (if (gethash "additionalTextEdits" item)
```

Comment:
Just asking, which language server that requires `completionItem/resolve` for `additionalTextEdits`?
From the specs, I saw that only `documentation` and `detail` are subjects of delay computation, can we perhaps drop the `lsp--resolve-completion` in this case entirely?

Reply:
Otherwise, the change LGTM

Reply:
JDT LS was populating the additionalTextEdits in resolve call but they fixed that behavior lately. I checked the spec and it seems like additionalTextEdits are not mentioned. The spec says:

```
The returned completion item should have the documentation property filled in. The request can delay the computation of the detail and documentation properties. However, properties that are needed for the initial sorting and filtering, like sortText, filterText, insertText, and textEdit must be provided in the textDocument/completion response and must not be changed during resolve.
```

The issue is that additonalTextEdit is not mentioned in any of the lists. I will ask it the lsp repo for clarification. It will be great if we can drop that resolve call.